server/src/main/java/io/spine/server/bus/BusBuilder.java

```diff
@@ -78,9 +79,9 @@ protected BusBuilder() {
      * <p>The order of appending the filters to the builder is the order of the filters in
      * the resulting bus.
      *
-     * @param filter the filter to append
+     * @param filter the filter to add
      */
-    public final B appendFilter(BusFilter<E> filter) {
+    public final B addFilter(BusFilter<E> filter) {
```

Comment:
Append seems more descriptive to me. While we can `add` elements to a set, `append` implies ordering.